src/Phinx/Migration/Manager.php

```diff
@@ -555,7 +609,7 @@ public function setMigrations(array $migrations)
      * Gets an array of the database migrations.
      *
      * @throws \InvalidArgumentException
-     * @return AbstractMigration[]
+     * @return AbstractMigration[], indexed by migration name (aka creation time) and sorted in ascending order
```

Comment:
`* @return AbstractMigration[] Indexed by migration name (aka creation time) and sorted in ascending order`

The return type and description are separate entities.